tools/log_formatter_test.go

```diff
@@ -42,6 +42,13 @@ func TestLogFormatter(t *testing.T) {
 			t.Errorf("expected '{d:1s}', got '%s'", lf.String())
 		}
 	})
+	t.Run("host", func(t *testing.T) {
```

Comment:
Thank you for writing this test! Can we also include `Host` in the "all" test [here](https://github.com/alpine-hodler/gidari/blob/main/tools/log_formatter_test.go#L68) ?

Reply:
Oh, missed that test..will do both the changes. Thankyou